benefits/core/templates/core/logged-out.html

```diff
@@ -7,12 +7,10 @@
 
 {% block main-content %}
   <div class="container">
-    <div class="row justify-content-lg-center">
+    <div class="row justify-content-center">
       <div class="col-md-6">
-        <h1 class="h2 text-center">
-          <span class="d-block pb-lg-8 pb-5">{% include "core/includes/icon.html" with name="happybus" %}</span>
-          {% translate "You have successfully logged out. Thank you for using Cal-ITP Benefits!" %}
-        </h1>
+        <div class="py-4 mt-4 text-center">{% include "core/includes/icon.html" with name="happybus" %}</div>
```

Comment:
I suggest `mt-5` here instead of `mt-4` to get 72px of spacing. This currently has us at 48px of spacing.

| Before (70px from `h1` rule) | Figma | This branch (48px) | Using `mt-5` |
| --- | --- | --- | --- |
| ![image](https://github.com/user-attachments/assets/34df5dcb-d3b4-43d8-a72a-e6adaf8a867d) | ![image](https://github.com/user-attachments/assets/52df7476-bf24-4af1-867a-7b2d425d2b3f) | ![Screenshot from 2024-12-10 13-11-46](https://github.com/user-attachments/assets/a583b759-7366-454b-b2a1-6cd9015aa9e1) | ![Screenshot from 2024-12-10 13-12-00](https://github.com/user-attachments/assets/cdef6f9d-aacc-4121-9ee5-81546d140f63) |

Reply:
Fixed: https://github.com/cal-itp/benefits/pull/2579/commits/279fc00f459b02effa6bb0316c937713d2948564